docs/src/pages/careers/full-stack-engineer.md

```diff
@@ -1,12 +1,11 @@
-# Full-stack Engineer - Toolpad
+# Full-stack Engineer - Toolpad (future role)
 
 <p class="description">You will join the MUI Toolpad team, to explore the role of MUI in the low code space and help bring the early prototype to a usable product.</p>
 
 ## Details of the Role
 
 - Location: Remote (preference for UTC-6 to UTC+5).
 - Type of work: Full-time (contractor or employee [depending on circumstances](https://mui-org.notion.site/Hiring-FAQ-64763b756ae44c37b47b081f98915501))
-- Start date: Immediately.
 - Level: [4 or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).
```

Comment:
Would this make sense? I'm not sure

```suggestion
- Start date: Once the next batch of roles is open.
- Level: [4 or above](https://docs.google.com/spreadsheets/d/1dDdPD-flNXlgZ0E3ZxVvCDx27RFuhVWJrcfcjNu_I8k/edit#gid=0).
```

Reply:
I thought we are removing this line altogether, for both the templates and the positions we have regardless if they are future or right now. Maybe I misunderstood it?
cc @mnajdova

Reply:
Ah my bad, I read too quickly. I thought the issue was that the line is not correct, that it was an opportunity to be more upfront. No objections to removing it from everywhere.

Reply:
@mnajdova @DanailH please move as you see fit, I don't have a preference. I have seen many companies not mention the start date, with the implicit idea that it's as soon as possible (probably better wording than immediately). If we feel that it's clear enough without it,  then 👍.